doc/whats_new/v1.1.rst

```diff
@@ -59,6 +59,21 @@ Changelog
 :mod:`sklearn.decomposition`
 ............................
 
+- |Fix| Expose a parameter `n_oversamples` to tune the `randomized_svd` and get
+  accurate results when the number of features is large.
+  :pr:`21109` by :user:`Smile <x-shadow-man>`.
+
+:mod:`sklearn.ensemble`
```

Comment:
You still have this entry that should be removed because it corresponds to another PR.
It is probably due to an issue when you solve `git` conflicts.